guides/common/attributes-titles.adoc

```diff
@@ -25,6 +25,7 @@
 // Release Notes - defined for Foreman and Katello
 //:ReleaseNotesDocTitle: Release Notes - base unused
 :TuningDocTitle: Tuning Performance of {ProjectName}
+:UpdatingDocTitle: Updating {ProjectName}
 :UpgradingDocTitle: Upgrading and Updating {ProjectName}
```

Comment:
Does this line need to be changed to just `Upgrading {ProjectName}`?

Reply:
Yes, but that will be managed by Akshay in another PR.

Reply:
I should've read the description again before commenting lol